src/views/FindOrder.vue

```diff
@@ -396,6 +396,25 @@ ion-modal {
   --border-radius: 8px;
 }
 
+main > div:hover {
```

Comment:
instead of cursor pointer, we should use an anchor tag here since this clicks out to another page

Reply:
As per our discussion, I have removed hover effect changes and only added cursor pointer to order div.